src/en/charms-storage-ceph.md

```diff
@@ -0,0 +1,123 @@
+Title: Installing Ceph
```

Comment:
would this work better as a tutorial on setting up a standalone ceph cluster with Juju?